chia/data_layer/data_layer.py

```diff
@@ -137,12 +137,14 @@ async def get_root(self, store_id: bytes32) -> Optional[bytes32]:
             self.log.error(f"Failed to get root for {store_id.hex()}")
         return res.node_hash
 
-    async def get_roots(self, store_ids: List[str]) -> List[Optional[bytes32]]:
+    async def get_roots(self, store_ids: List[str]) -> List[Dict[str, Any]]:
         roots = []
         for id in store_ids:
-            res = await self.data_store.get_tree_root(tree_id=bytes32(hexstr_to_bytes(id)))
+            id_bytes = bytes32(hexstr_to_bytes(id))
+            res = await self.data_store.get_tree_root(tree_id=id_bytes)
             if res is None:
```

Comment:
`.get_tree_root()` doesn't return `None`.  So this handling was already broken...  I did ask about why mypy didn't warn about this and it seems like it might be a bug as similar scenarios but with variables for example can be warned about.  I need to follow up on that and find or file an issue.